utils/controller.js

```diff
@@ -1,16 +1,18 @@
-const { models } = require('../dal/database');
+const { models, connectDb } = require('../dal/database');
```

Comment:
We should keep the DAL access within a handler.

Reply:
Do you think it would be a good idea to include the `getSimilarCategory` method inside of controller.js instead as a utility method?

Reply:
Yes.